docs/data/sdks/flutter/index.md

```diff
@@ -59,7 +59,39 @@ class YourClass {
 }
 ```
 
-### EU data residency
+### Configuration
+
+Amplitude Flutter SDK runs on the top of the [Amplitude Android Mantaince SDK](../android/), [Amplitude iOS Mantaince SDK](../ios/) and [Amplitude JavaScript Mantaince SDK](../javascript/). The following are the Dart settable config options. For more configurations on Android side, please check the [Android Configuration](../android-kotlin/#configuration). For more configurations on iOS side, please check the [iOS configuration](../ios/#configuration). For more configurations on browser side, please check the [browser configuration](../javascript/#configuration).
+
+???config "Configuration Options"
+    | <div class="big-column">Name</div>  | Description | Default Value |
+    | --- | --- | --- |
+    | `enableCoppaControl` | Enable COPPA (Children's Online Privacy Protection Act) restrictions on IDFA, IDFV, city, IP address and location tracking.| By default, coppa control is disabled. |
+    | `disableCoppaControl` | Disable COPPA (Children's Online Privacy Protection Act) restrictions on IDFA, IDFV, city, IP address and location tracking.|  By default, coppa control is disabled. |
+    | `setMinTimeBetweenSessionsMillis` | `int`. The amount of time for session timeout if disable foreground tracking. | `5 minutes` |
+    | `setEventUploadThreshold` | `int`. SDK will attempt to upload once unsent event count exceeds the event upload threshold or reach the 30 seconds. | `30` |
+    | `setServerZone` | `String`. The server zone to send to, will adjust server url based on this config. | `US` |
+    | `setServerUrl` | `String`. The API endpoint URL that events are sent to. Automatically selected by `ServerZone` | `https://api2.amplitude.com/` |
+    | `setUseDynamicConfig` | `bool`. Find the best server url automatically based on users' geo location. | `false` |
+    | `setOptOut` | `bool`. Opt the user out of tracking. | `false` |
+    | `trackingSessionEvents` | `bool`. Flushing of unsent events on app close. | `false` |
+    | `useAppSetIdForDeviceId` | Only for Android. Whether to use app ser id as device id on Android side. Need to include the module and permission. | By default, the deviceId will be UUID+"R" |
+    
+#### Configure batching behavior
+
+To support high performance environments, the SDK sends events in batches. Every event logged by `logEvent` method is queued in memory. Events are flushed in batch in background. You can customize batch behavior with `setEventUploadThreshold`. By default, the serverUrl will be `https://api2.amplitude.com/`. This SDK doesn't support batch mode, the [batch API](../../../analytics/apis/batch-event-upload-api.md) endpoint.
```

Comment:
Can users use `setServerUrl` to batch API endpoint as setting to batch mode?

Reply:
I think we can. but there is no use of the batch method to auto-configure the endpoint to the batch API. I will re-phrase this.